apps/backoffice-v2/src/pages/TransactionMonitoringAlerts/components/AlertsTable/hooks/useAlertsTableLogic/useAlertsTableLogic.tsx

```diff
@@ -21,21 +25,49 @@ export const useAlertsTableLogic = ({ data }: { data: TAlertsList }) => {
   ]);
   const onSortingChange: OnChangeFn<SortingState> = useCallback(
     updaterOrValue => {
```

Comment:
can you change the name to `sortingUpdater`?

Reply:
You mean something like this?
```suggestion
    sortingUpdaterOrValue => {
```